contexts/network-settings.tsx

```diff
@@ -473,19 +475,19 @@ export const NetworkSettingsProvider = ({ children }) => {
   }
 
   useEffect(() => {
-    if (!network || !state.Service?.active || state.Service?.starting)
+    if (!network || !daoService || state.Service?.starting)
```

Comment:
isn't `state?.Service.starting` something that should be inside the `useDaoService` ?